api/api/error_handler.py

```diff
@@ -75,7 +75,7 @@ async def expect_failed_error_handler(request: ConnexionRequest, exc: ExpectFail
         problem['detail'] = exc.detail
 
     return json_response(data=problem, pretty=request.query_params.get('pretty', 'false') == 'true',
-                         status_code=exc.status, content_type=ERROR_CONTENT_TYPE)
+                         status_code=exc.status)
```

Comment:
I would keep returning problem+json in all errors.

Reply:
The test was failing due to the spec not containing the `application/xml` content-type when `raw` was used. Due to this error, the logs showed an unexpected `application/problem+json` content-type that I thought were conflicting with the test but they aren't.